### PR TITLE
kubeadm:return error info correctly

### PR DESCRIPTION
**Pull request by navist2020**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
https://github.com/kubernetes/kubernetes/blob/b496238dd65d86c65183ac7ffa128c5de46705b4/cmd/kubeadm/app/cmd/token.go#L281-L283
Line282: Normally, the first parameter `err` will be nil and this causes no error to be returned
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by pacoxu** [comment]
It would be a problem.
Can this problem be easily reproduced with a command?

/lgtm
/assign @sataqiu
/ok-to-test

**Comment by navist2020** [comment]
Hi @pacoxu 
> Can this problem be easily reproduced with a command?

When I use --certificate-key without --print-join-command like:
`kubeadm token create --certificate-key=xxxxx`
I didn't get any warnings.But I think the error should have been returned here.

**Review by neolit123 — COMMENTED** [review]
thanks
/lgtm
/approve
/triage accepted

File: cmd/kubeadm/app/cmd/token.go [hunk 1]
```diff
@@ -279,7 +279,7 @@ func RunCreateToken(out io.Writer, client clientset.Interface, cfgPath string, i
 		}
 	} else {
 		if certificateKey != "" {
-			return errors.Wrap(err, "cannot use --certificate-key without --print-join-command")
+			return errors.New("cannot use --certificate-key without --print-join-command")
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/token.go:282]: seems like a copy / paste mistake.

**Pull request merged**